metadata/UiPathDocumentOCR__6__metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "changeLog": "Model: 21.10.1",
+  "cpu": 0,
+  "description": "Machine Learning model for extracting text and checkboxes/radio buttons from Documents. Please see more details including supported languages and link to Activities guide in the Document Understanding Guide here: https://docs.uipath.com/document-understanding/v2021.4/docs/ml-packages#uipath-document-ocr. For alternative deployment methods in high-volume scenarios see also this page: https://docs.uipath.com/document-understanding/v2021.4/docs/ocr-services",
```

Comment:
Should we delete OCR 5? Otherwise we will see 2 OCE entries?

Reply:
Maybe we should update the existing 5 version. Would it be any issue? @vivekjhauipath

Reply:
I don't see any issue, @irinel-birsan-uipath in prod does 21.10 model point to v5?

Reply:
@irinel-birsan-uipath we have too many dococr models in prod - should this one be 9 to match latest?
![image](https://user-images.githubusercontent.com/45358096/135993222-ebd3f272-10d4-4cfd-b9d8-01503c921a04.png)

Reply:
Yes, it matches the V9 from prod europe.

Reply:
I don't think we have to match the cloud versions as they are different based on the region.